data/sql/updates/pending_db_world/rev_1556033308686302400.sql

```diff
@@ -0,0 +1,232 @@
+(17610, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 11, 31734, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Agent - On Aggro - Cast \'31734\''),
+(17610, 0, 1, 0, 0, 0, 100, 0, 1700, 1700, 8900, 8900, 11, 14873, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Agent - In Combat - Cast \'14873\''),
+(17610, 0, 2, 0, 0, 0, 100, 0, 12400, 12400, 38600, 38600, 11, 15691, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Agent - In Combat - Cast \'15691\'');
+
+
+-- Creating a new SmartAI script for [Creature] ENTRY 17609 (name: Sunhawk Saboteur)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 17609;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 17609);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(17609, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 11, 31734, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Saboteur - On Aggro - Cast \'31734\''),
+(17609, 0, 1, 0, 0, 0, 100, 0, 1000, 1000, 6200, 6200, 11, 6660, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Saboteur - In Combat - Cast \'6660\''),
+(17609, 0, 2, 0, 0, 0, 100, 0, 11750, 11750, 32400, 32400, 11, 14443, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Saboteur - In Combat - Cast \'14443\'');
+
+
+-- Creating a new SmartAI script for [Creature] ENTRY 17322 (name: Infected Wildkin)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 17322;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 17322);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(17322, 0, 0, 0, 0, 0, 100, 0, 1500, 1500, 3200, 3200, 11, 31282, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Infected Wildkin - In Combat - Cast \'31282\'');
```

Comment:
This creature ENTRY 17322 (name: Infected Wildkin) doesn't have any abilities, so you remove it